pymc3/distributions/multivariate.py

```diff
@@ -38,6 +38,11 @@
 from .shape_utils import to_tuple
 from ..math import kron_dot, kron_diag, kron_solve_lower, kronecker
 
+try:
```

Comment:
I would add a note that this should be deprecated once we are all the way over to the new theano.

Reply:
We could just make a new release for Theano-PyMC and update the version in `setup.py`, then we wouldn't even need the `try` statement.  Want to do that instead?

Reply:
I'd rather stay compatible with Theano proper for a little longer.

Reply:
OK, I added a `TODO` comment.